components/match2/commons/starcraft_starcraft2/match_group_input_starcraft_ffa.lua

```diff
@@ -181,6 +181,7 @@ function StarcraftFfaInput._matchWinnerProcessing(match, numberOfOpponents, nosc
 		--determine MATCH winner, resulttype and walkover
 		if walkover ~= '' then
```

Comment:
```suggestion
		if walkover then
```
 + remove the `or ''` at L176
should remove the need for disable diagnostic
Alternatively, change'
```
local walkover = match.walkover or ''
``` 
to 
```
---@type string|number
local walkover = match.walkover or ''
```

Reply:
nope (to both), still says number is not string:
![Screenshot 2023-05-16 10 34 18](https://github.com/Liquipedia/Lua-Modules/assets/75081997/0609e70a-1036-491e-8142-a5c2e7aa4fab)

Reply:
circumvented it with just using another local var